docs/modeling/configuring.rst

```diff
@@ -5,6 +5,7 @@ Entity Framework uses a set of :doc:`default-conventions` to build a model based
 
 .. note::
     This article covers configuration that can be applied to a model targeting any data store and that which can be applied when targeting any relational database.
+
     Providers may also enable configuration that is specific to a particular data store. For documentation on provider specific configuration see the the :doc:`/providers/index` section.
 
 In this article:
```

Comment:
Suggestion: we could use the `.. contents::` rst helper to automate these links:

```rst
.. contents:: In this article:
   :depth: 2
```